src/Utils/RemovableInputField.js

```diff
@@ -1,16 +1,29 @@
 import React, { Component } from 'react';
 
-class InputField extends Component {
+class RemovableInputField extends Component {
```

Comment:
It is a lot more descriptive name. Not every input field should be removable. that's the speciality about this input field. That's why a new name.